### ISSUE TITLE
Negative upload results when up BW is somehow high (~500Mbps)

### ISSUE THREAD

**Issue by crbertoldo** [showwin/speedtest-go#88]
Hi, looks like https://github.com/showwin/speedtest-go/commit/dafff30f5a9e3d498961c9b203c195ccc5721a27 tried to fix it but it seems we still have negative values for the upload test when the up BW is too high (~500 Mbps per the Speedtest CLI tool by Ookla - which, BTW, gives higher values for both up and down). Tested both in 1.1.5 and 1.3.0. Note: `--saving-mode` gives lower values (~50%), but positive ones.

```
support@host:~$ ./speedtest-go --version
1.3.0
```
```
support@host:~$ ./speedtest-go
Testing From IP: *.*.*.*, (CenturyLink) [*, *] 
Target Server: [52079]    89.06km Denver, CO (United States) by CenturyLink, Inc.
Latency: 2.5150499s
Download Test: ......
Upload Test: .
 
Download: 448.59 Mbit/s
Upload: -6.55 Mbit/s
```
```
support@host:~$ ./speedtest-go --saving-mode
Testing From IP: *.*.*.*, (CenturyLink) [*, *] 
 
Target Server: [52079]    89.06km Denver, CO (United States) by CenturyLink, Inc.
Latency: 2.510973709s
Download Test: ......
Upload Test: .
 
Download: 479.07 Mbit/s
Upload: 222.53 Mbit/s
```

**Comment by crbertoldo** [comment]
@r3inbowari @showwin, thank you for https://github.com/showwin/speedtest-go/pull/89.
```
support@host:~/speedtest-go$ ./speedtest-go
Testing From IP: *.*.*.*, (CenturyLink) [*, *] 
 
Target Server: [52079]    89.06km Denver, CO (United States) by CenturyLink, Inc.
Latency: 2.516662631s
Download Test: ......
Upload Test: ....
 
Download: 448.63 Mbit/s
Upload: 404.37 Mbit/s
```
Could 1.3.1 be released for it?

**Comment by showwin** [comment]
@crbertoldo I've just released 1.3.1 which includes the fix 👍 
https://github.com/showwin/speedtest-go/releases/tag/v1.3.1

@r3inbowari Thanks again!

**Comment by crbertoldo** [comment]
> @crbertoldo I've just released 1.3.1 which includes the fix 👍 https://github.com/showwin/speedtest-go/releases/tag/v1.3.1
> 
> @r3inbowari Thanks again!

Thanks a lot!

**Comment by r3inbowari** [comment]
Yes, I will add a flag `--icmp` later
